odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -88,11 +90,24 @@ def test_rpcstyle_one_by_one(self):
         for name in range(10):
             self.multi.lines = [(0, 0, {"name": str(name)})]
         self.operations()
+        self.multi.unlink()
+        self.multi = self.env["test_new_api.multi"].new({
+            "name": "What is up?"
+        })
+        for name in range(10):
+            self.multi.lines = [(0, 0, {"name": str(name)})]
+        self.operations(True)
 
     def test_rpcstyle_single(self):
         """Check lines created with RPC style and added in one step"""
         self.multi.lines = [(0, 0, {'name': str(name)}) for name in range(10)]
         self.operations()
+        self.multi.unlink()
+        self.multi = self.env["test_new_api.multi"].new({
+            "name": "What is up?"
+        })
+        self.multi.lines = [(0, 0, {'name': str(name)}) for name in range(10)]
+        self.operations(True)
```

Comment:
Please make a new test method test_rpcstyle_single_on_new with the 5 lines above.